pkg/buildscript/mutations.go

```diff
@@ -125,7 +141,7 @@ func (b *BuildScript) UpdatePlatform(operation types.Operation, platformID strfm
 	return nil
 }
 
-func (b *BuildScript) addPlatform(platformID strfmt.UUID) error {
+func (b *BuildScript) AddPlatform(platformID strfmt.UUID) error {
```

Comment:
By changing the visibility of this method, it's possible that `UpdatePlatform()` is no longer needed. Please consider removing it.